### PR TITLE
Fix `hide-tips` CSS feature

### PR DESCRIPTION
**Pull request by cheap-glitch**
## Test URLs

 * [PR list](https://github.com/refined-github/refined-github/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc)
 * [Gobal conversations list](https://github.com/issues?q=is%3Aopen+author%3A%40me+archived%3Afalse+sort%3Aupdated-desc)

## Screenshot

**Before**

screenshot-1639343654

**After**

screenshot-1639343639

### CODE REVIEW

File: source/features/hide-tips.css [hunk 1]
```diff
@@ -23,8 +23,8 @@ a.tabnav-extra[href$='mastering-markdown/'] {
 	display: none !important;
 }
 
-/* Remove random protip at the bottom of the page */
-.protip {
+/* Remove random protip at the bottom of conversations lists */
+.paginate-container + .text-center:last-child {
```
Comment (cheap-glitch) [inline source/features/hide-tips.css:27]: I haven't seen those tips anywhere else, apart from one at the very bottom of the dashboard (not really worth fixing, plus its DOM is annoying).

Also, worth keeping the old selector for GHE?

Comment (fregante) [reply source/features/hide-tips.css:27]: > Also, worth keeping the old selector for GHE?

No

**Comment by fregante** [comment]
# ขอบคุณครับ

**Comment by kidonng** [comment]
There's also a tip on the dashboard:



However I'm not sure if we want to hide it since the links in it are useful (?)

**Comment by cheap-glitch** [comment]
> There's also a tip on the dashboard (above the square, too lazy to make another screenshot):

https://github.com/refined-github/refined-github/pull/5196#discussion_r767328562

**Pull request merged**